version.php

```diff
@@ -25,8 +25,8 @@
 defined('MOODLE_INTERNAL') || die();
 
 $plugin->component = 'mod_zoom';
-$plugin->version = 20250408000;
-$plugin->release = 'v2.6.2';
+$plugin->version = 20250417000;
+$plugin->release = 'v3.0.0';
```

Comment:
@Yashbanu shouldn't this qualify for a patch release 2.6.3?

Reply:
@meyyappanv logically in relation with changes in this PR increasing the patch version make sense.

But In past we have done multiple changes like registration for meeting, fix for multiple instance of zoom meeting while fetching recoding and even rebased linkstreet/stable from master branch but missed to update release version since 2020. 

Thats why I asked her to create tag is 3.0.0.  

If you still feel we should stick to patch version update then @Yashbanu can stick to that.